### PR TITLE
closeSend and fix flakytest

### PR DESCRIPTION
**Pull request by irisdingbj**

### CODE REVIEW

**Comment by irisdingbj** [comment]
@linsun

File: pilot/pkg/proxy/envoy/v2/ads_test.go [hunk 1]
```diff
@@ -160,18 +160,19 @@ func TestAdsClusterUpdate(t *testing.T) {
...
 
 	// register a second endpoint
-	_ = server.EnvoyXdsServer.MemRegistry.AddEndpoint("adsupdate2.default.svc.cluster.local",
+	_ = server.EnvoyXdsServer.MemRegistry.AddEndpoint("adsclusterupdate2.default.svc.cluster.local",
 		"http-status", 2080, "10.2.0.2", 1081)
 
-	cluster2 := "outbound|80||adsupdate2.default.svc.cluster.local"
+	cluster2 := "outbound|80||adsclusterupdate2.default.svc.cluster.local"
 	sendEDSReqAndVerify(cluster2)
+	edsstr.CloseSend()
```
Comment (hzxuzhonghu) [inline pilot/pkg/proxy/envoy/v2/ads_test.go]: You donot need this, I have a pr in release-1.1 #10003

**Comment by linsun** [comment]
@irisdingbj thanks, could you rebase this PR on 1.1 if it is needed after #10003?

**Comment by irisdingbj** [comment]
@linsun Let me resubmit a new pr for 1.1 to address the issue.

**Pull request closed without merging**